File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/comment/direct/Java/Go/s466963472_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func solve(d int) int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Split(bufio.ScanWords)
	for sc.Scan() {
		token := sc.Text()
		d, err := strconv.Atoi(token)
		if err != nil {
			continue
		}
		fmt.Println(solve(d))
	}
}

// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/comment/direct/Java/Python/s466963472_commented.py
import sys

def solve(d):
    ans = 0
    for i in range(1, (600 // d)):
        x = i * d
        ans += x * x * d
    return ans

def main():
    data = map(int, sys.stdin.read().split())
    for d in data:
        print(solve(d))

if __name__ == "__main__":
    main()
# End of Code
